Mathlib/Algebra/Polynomial/Splits.lean

```diff
@@ -430,7 +403,7 @@ theorem splits_id_of_splits {f : K[X]} (h : Splits (f.map i))
 
 theorem splits_comp_of_splits (i : R →+* K) (j : K →+* L) {f : R[X]} (h : Splits (f.map i)) :
     Splits (f.map (j.comp i)) :=
-  (splits_map_iff i j).mp (splits_of_splits_id _ <| (splits_map_iff i <| .id K).mpr h)
+  f.map_map i j ▸ h.map j
```

Comment:
I'm a little conflicted about keeping this lemma since it only talks about the `Splits` of `map`. But it does match the direction of `Polynomial.map_map` (even if that isn't quite a `@[simp]` lemma). So I think it is good to keep.

How about renaming this to `Splits.map_comp`?